modules/extension/xsd/xsd-wfs/src/test/java/org/geotools/wfs/v2_0/bindings/DescribeStoredQueriesTypeBindingTest.java

```diff
@@ -56,7 +56,7 @@ public void testParse() throws Exception {
         assertNotNull(desc);
         
         assertEquals("Hirlam Pressure Grid", desc.getTitle().get(0).getValue());
-        assertEquals("Hirlam forecast model's pressure levels as a grid data encoded in GRIB format.", desc.getAbstract().get(0).getValue());
+        assertEquals("\n            Hirlam forecast model's pressure levels as a grid data encoded in GRIB format.\n        ", desc.getAbstract().get(0).getValue());
```

Comment:
This does not seem right, the XML does not have a CDATA section, this text should be trimmed:

````
        <Title>Hirlam Pressure Grid</Title>
        <Abstract>
            Hirlam forecast model&apos;s pressure levels as a grid data encoded in GRIB format.
        </Abstract>
````

Reply:
The XML spec actually says that elements extending xs:String should preserve all whitespace. See https://www.w3.org/TR/xmlschema-2/#dt-whiteSpace for more details.

Reply:
Indeed I'm reading the same, but find it surprising :-)
@bencaradocdavies am I missing something here, or is indeed whitespace meant to be preserved there?